notebooks/growth_model_eda.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>Thanks for adding the nb Hema.  :)</p><p> </p><p>Here are some suggestions on filtering:</p><p> </p><p>Could not make changes to this fork.. </p><p>Adding some lines here... </p><p> </p><pre class="ql-syntax" spellcheck="false">grouped = df_filtered.groupby('time_series_code')
#filter only those containing 39 unique dates
filtered_groups = grouped.filter(lambda x: x['date'].nunique() == 39)
# get the list of time_series_code with 39 dates
selected_time_series = filtered_groups['time_series_code'].unique()
#filtr the org df
filtered_df = df_filtered[df_filtered['time_series_code'].isin(selected_time_series)]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Thanks for the suggestions Surya. I made somewhat similar changes to have this logic included, kindly please review the new changes and let me know if it looks good